acme_diags/libs/README.md

```diff
@@ -0,0 +1,13 @@
+# Wrapped Libraries
```

Comment:
Added a README to the `acme_diags/libs` directory to make its purpose clear and to provide a usage example.

Reply:
Very nice!